src/test/java/ro/unibuc/triplea/domain/reviews/steam/service/SteamGameReviewServiceTest.java

```diff
@@ -0,0 +1,243 @@
+package ro.unibuc.triplea.domain.reviews.steam.service;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import ro.unibuc.triplea.application.games.steam.dto.response.SteamGameResponse;
+import ro.unibuc.triplea.application.reviews.steam.dto.request.SteamGameReviewRequest;
+import ro.unibuc.triplea.application.reviews.steam.dto.response.SteamGameReviewResponse;
+import ro.unibuc.triplea.domain.games.steam.exception.SteamGameNotFoundException;
+import ro.unibuc.triplea.domain.games.steam.service.SteamGameService;
+import ro.unibuc.triplea.domain.reviews.steam.model.entity.SteamGameReview;
+import ro.unibuc.triplea.domain.reviews.steam.repository.SteamGameReviewRepository;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.*;
+
+public class SteamGameReviewServiceTest {
+
+    private SteamGameReviewService steamGameReviewService;
+
+    @Mock
+    private SteamGameReviewRepository steamGameReviewRepository;
+
+    @Mock
+    private SteamGameService steamGameService;
+
+    @BeforeEach
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
Done. It was for another way of initializing mocks: you can either initialize them through for example `MyClass mocked = Mockito.mock(MyClass.class`), our you can simply use the `@Mock` annotation, then use this setup code. Since we don't have too many mocks anyway, I think the first style is better in our case though. I mixed the two up; so I removed the `setUp()`